### PR TITLE
[BACKLOG-30711] Hadoop Cluster UX - Update Cluster Test Results screen to use Thin dialog

### PR DESCRIPTION
**Pull request by e-cuellar**

### CODE REVIEW

**Comment by buildguy** [comment]
### :gear: Build finished in 55s with command:
```
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -pl kettle-plugins/hadoop-cluster,kettle-plugins/hadoop-cluster/ui
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|3|0|0|

|[Details](http://wingman2.pentaho.net/job/projects/job/big-data-plugin/job/PR-1782/1/testReport)|
|----|

:information_source:  _This is an automatic message_

File: kettle-plugins/hadoop-cluster/pom.xml [hunk 1]
```diff
@@ -57,6 +57,12 @@
         <version>${pdi.version}</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
+        <groupId>pentaho</groupId>
+        <artifactId>pentaho-big-data-impl-clusterTests</artifactId>
+        <version>${project.version}</version>
+        <scope>provided</scope>
```
Comment (ricksjames) [inline kettle-plugins/hadoop-cluster/pom.xml]: should this be test scope

File: kettle-plugins/hadoop-cluster/ui/pom.xml [hunk 2]
```diff
@@ -68,6 +68,10 @@
       <groupId>pentaho</groupId>
       <artifactId>pentaho-i18n-webservice-bundle</artifactId>
     </dependency>
+    <dependency>
+      <groupId>pentaho</groupId>
+      <artifactId>pentaho-big-data-impl-clusterTests</artifactId>
```
Comment (ricksjames) [inline kettle-plugins/hadoop-cluster/ui/pom.xml]: should this be test scope

File: kettle-plugins/hadoop-cluster/pom.xml (hunk 1 above)
Comment (e-cuellar) [reply kettle-plugins/hadoop-cluster/pom.xml]: Done!

File: kettle-plugins/hadoop-cluster/ui/pom.xml (hunk 2 above)
Comment (e-cuellar) [reply kettle-plugins/hadoop-cluster/ui/pom.xml]: I think this one is inherited from its parent....

**Comment by buildguy** [comment]
### :gear: Build finished in 53s with command:
```
mvn clean verify -B -e -Daether.connector.resumeDownloads=false -amd -Daudit -P !assemblies -pl kettle-plugins/hadoop-cluster,kettle-plugins/hadoop-cluster/ui
```
:ok_hand: All tests passed!
 ---

|Total|Failed|Skipped|
|---|---|---|
|3|0|0|

|[Details](http://wingman2.pentaho.net/job/projects/job/big-data-plugin/job/PR-1782/4/testReport)|
|----|

:information_source:  _This is an automatic message_

**Pull request merged**